content/en/apps/examples/anc.md

```diff
@@ -1,16 +1,55 @@
 ---
-title: "Antenatal Care Reference App"
-linkTitle: "ANC Reference App"
+title: "Reference Application: ANC"
+linkTitle: "Reference Application: ANC"
```

Comment:
A reference app isn't a term that is consistently used in our field, so I am hesitant to make it so prominent in the title here. The term is used with some tools, whereas others use other terms, like "Templates". From an external perspective, what is the difference between an example and reference app? Do we need to make that distinction in the menu structure, or just clarify early in the text what a reference app is? Eg that this example is ready to use as-is and has full set of tests.

Reply:
Yeah, I was unsure about that as well. Referring to it similarly to the other examples- "Antenatal Care" - with an explanation that it's a reference app, makes sense.